features/call/impl/src/main/kotlin/io/element/android/features/call/impl/utils/WebViewWidgetMessageInterceptor.kt

```diff
@@ -117,6 +123,15 @@ class WebViewWidgetMessageInterceptor(
 
                 super.onReceivedSslError(view, handler, error)
             }
+
+            override fun shouldInterceptRequest(view: WebView?, request: WebResourceRequest): WebResourceResponse? {
+                println("Intercepting: ${request.url}")
+                return assetLoader.shouldInterceptRequest(request.url)
+            }
+
+            override fun shouldInterceptRequest(view: WebView?, url: String): WebResourceResponse? {
+                return assetLoader.shouldInterceptRequest(Uri.parse(url))
```

Comment:
Will this work when the Element Call url is tweaked by the user (`getCustomElementCallBaseUrlFlow`)?

Reply:
Yes, it will return `null`, which will tell the `WebView` there's no need to intercept the request.